### ISSUE TITLE
What is the better/latest way to develop app using storyboard or xib files

### ISSUE THREAD

**Issue by arvindrajachourasiya** [MicrosoftDocs/xamarin-docs#949]
I wonder what is better/recommended way to create application in iOS using storyboards or xib files? As I heard using .xib we should develop the app. I am working with old xamarin.iOS app totally developed using xib.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5885483e-90ee-12e6-4c3f-0a403df7eb88
* Version Independent ID: d44e686f-ece1-e733-845c-5dda604b4927
* Content: [Introduction to Storyboards in Xamarin.iOS - Xamarin](https://docs.microsoft.com/en-us/xamarin/ios/user-interface/storyboards/?tabs=windows)
* Content Source: [docs/ios/user-interface/storyboards/index.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/ios/user-interface/storyboards/index.md)
* Product: **xamarin**
* GitHub Login: @lobrien
* Microsoft Alias: **laobri**

**Comment by arvindrajachourasiya** [comment]
@remmah - Thank you for useful information. Long ago when I started working on Xamarin one of my senior who already was working on Xam.iOS project , said that using .xib we have more benefits like multiple people can work on a single project. When it comes to storyboard we single file so only one person can have access at a time. Since than I moved towards .xib implimentatio but when I seen more things like stackoverflow question & this docs every one talks about stroyboards than I raised this question.
